contracts/modules/ApprovedTransfer.sol

```diff
@@ -100,6 +115,27 @@ contract ApprovedTransfer is BaseTransfer {
         onlyAuthorisedContractCall(_wallet, _contract)
     {
         doApproveTokenAndCallContract(_wallet, _token, _spender, _amount, _contract, _data);
+        doResetLimit(_wallet);
+    }
+
+    /**
+     * @dev Changes the daily limit. The change is immediate.
+     * @param _wallet The target wallet.
+     * @param _newLimit The new limit.
+     */
+    function changeLimit(address _wallet, uint256 _newLimit) external onlyWalletModule(_wallet) onlyWhenUnlocked(_wallet) {
+        limitStorage.setLimit(_wallet, ILimitStorage.Limit(LimitUtils.safe128(_newLimit), uint128(0), uint64(0)));
+        doResetLimit(_wallet);
+        // solium-disable-next-line security/no-block-members
+        emit LimitChanged(_wallet, _newLimit, uint64(now));
+    }
+
+    /**
+    * @dev Resets the daily consumtion, i.e. DailySpent.alreadySpent = 0 and DailySpent.periodEnd = now.
+    * @param _wallet The target wallet.
+    */
+    function resetLimit(address _wallet) external onlyWalletModule(_wallet) onlyWhenUnlocked(_wallet) {
```

Comment:
Please rename as it's confusing the "daily spent" with "daily limit" which are two different values in limit storage. Maybe call it resetDailySpent?

Reply:
Done.